src/parser/line.rs

```diff
@@ -118,15 +119,15 @@ impl<'l> Line<'l> {
 
 impl<'l> From<&'l str> for Line<'l> {
     fn from(line: &'l str) -> Self {
-        if line.is_empty() || line.starts_with(HASH) {
+        if line.is_empty() || line.starts_with(HASH) || line.eq(EXPORT) {
             return Self::Empty;
         };
 
         let mut parts = line.splitn(2, '=');
 
         match (parts.next(), parts.next()) {
             (Some(k), Some(v)) => {
-                let key = k.trim();
+                let key = k.strip_prefix(EXPORT).unwrap_or(k).trim();
```

Comment:
```suggestion
                let key = k.strip_prefix(concat!(EXPORT, " ")).unwrap_or(k).trim();
```

Reply:
I missed this last time. But `strip_prefix` would also strip `export` regardless of whether the variable is exported or not like `exportBASIC=basic` would be `{ key: "BASIC", value: "basic" }`. We can add a space while stripping which might prevent this.

Reply:
I presumed the trim would take care of spaces, but never thought of the case `exportBASIC`. Good catch.

The suggestion to concat, however, is invalid rust. as concat only accepts literals as its arguments. I've played around with `format!` or `+`, but all of these require a String and then `to_owned()` or borrowing at some point. Making the code more complex than I'm confortable with.
So I went with the simplest solution, an additional constant resembling an export-with-whitespace. It's not ideal, but works. I'm too new to Rust, to find a nicer solution. Please feel free to correct me, as I'm sure there's a cleaner and nicer way somewhere. 

I've made that into a separate commit, so it can be improved upon (or reverted).

Reply:
> So I went with the simplest solution, an additional constant resembling an export-with-whitespace.

That's fine by me :)